Pack-Smart/pack-smart-fe
# Feature/edit packing list

## What's this PR do?
- Add functionality to items that also updates Redux Store:
     * User can check items on StaticItem component
     * User can update quantity of items StaticItem component
     * User can delete items via a modal in PackList component
     * Use can opt out of modal and delete items directly from StaticItem component 

## Where should the reviewer start?
- PackingList and StaticItem, as well as reducer.js and actions.js

## How should this be manually tested?
- Fill out the quiz and click submit
- On the packing list: 
    * Check items
    * change quantity
    * delete items
- Check Redux dev tools to see that the redux store has been updated.

## What are the relevant tickets?
- Closes #18 
- Closes #30 
- Closes #34 

## Screenshots/GIFs
- ![Screen Shot 2021-02-24 at 8 45 11 AM](https://user-images.githubusercontent.com/62810592/109026153-a9b27800-767c-11eb-9d59-02dc7d92dc94.png)

## Questions:
- We had talked about possibly moving the modal to it's own component. We are also planning to make a modal for adding an item. Perhaps it can be re-used? (note, ask Kara!)
- We have talked about a fancier input for changing item quantities. note: ask Alyssa about the npm package!
- We changed the edit pencil by the categories to a `+ `sign. note: ask JP about a message appearing on hover: 'Add item to category' or something similar.


## Did you update the README?
- No.
